osu.Game.Tests/Visual/Gameplay/TestSceneSkinnableHUDOverlay.cs

```diff
@@ -43,7 +42,7 @@ public partial class TestSceneSkinnableHUDOverlay : SkinnableTestScene
 
         // best way to check without exposing.
         private Drawable hideTarget => hudOverlay.KeyCounter;
-        private FillFlowContainer<KeyCounter> keyCounterFlow => hudOverlay.KeyCounter.ChildrenOfType<FillFlowContainer<KeyCounter>>().First();
+        private Drawable keyCounterFlow => (Drawable)hudOverlay.KeyCounter.Counters;
```

Comment:
This is the dodgiest code I've seen in a *long time*. To the point my brain has trouble comprehending how it even compiles..